language_service/src/hover.rs

```diff
@@ -158,12 +188,24 @@ impl<'a> Handler<'a> for HoverGenerator<'a> {
         &mut self,
         path: &'a ast::Path,
         _: &'_ hir::ItemId,
-        _: &'a hir::Package,
+        item: &'a hir::Item,
```

Comment:
Is there really a need to add this new parameter here? We can easily use the existing `ItemId` passed in right?

```rust
        let item = package
            .items
            .get(item_id.item)
            .expect("item id should exist");
```

Sure it may mean that we're doing one extra lookup, but I think I'd rather trade that off for keeping these signatures clean. They're pretty subtle as they are.

Reply:
It's just that we already do that exact lookup (and have to do it) prior to calling these functions: https://github.com/microsoft/qsharp/blob/6ff58ff16ee80285522e9856132e186db2cff54c/language_service/src/name_locator.rs#L393

Reply:
Seeing as how all the other features' uses of these functions only need the Item Id, it would be great to eliminate the extra params.

Reply:
done